pkg/installmanager/installmanager.go

```diff
@@ -705,75 +735,27 @@ func (m *InstallManager) gatherClusterLogs(cd *hivev1.ClusterDeployment) error {
 	return err
 }
 
-func (m *InstallManager) gatherBootstrapNodeLogs(cd *hivev1.ClusterDeployment) error {
-
-	m.log.Info("attempting to gather logs from cluster bootstrap node")
-
+func (m *InstallManager) initSSHKey() (string, error) {
 	m.log.Debug("checking for SSH private key")
 	sshPrivKeyPath := os.Getenv("SSH_PRIV_KEY_PATH")
 	if sshPrivKeyPath == "" {
-		m.log.Warn("cannot gather logs as SSH_PRIV_KEY_PATH is unset or empty")
-		return fmt.Errorf("cannot gather logs as SSH_PRIV_KEY_PATH is unset or empty")
+		m.log.Warn("cannot configure SSH agent as SSH_PRIV_KEY_PATH is unset or empty")
+		return "", fmt.Errorf("cannot configure SSH agent as SSH_PRIV_KEY_PATH is unset or empty")
 	}
 	fileInfo, err := os.Stat(sshPrivKeyPath)
 	if err != nil && os.IsNotExist(err) {
-		m.log.Warn("cannot gather logs/tarball as no ssh private key file found")
-		return err
+		m.log.WithField("path", sshPrivKeyPath).Warn("SSH_PRIV_KEY_PATH defined but file does not exist")
+		return "", err
 	} else if err != nil {
-		m.log.WithError(err).Error("error stating file containing private key")
-		return err
-	}
-
-	newSSHPrivKeyPath, err := m.chmodSSHPrivateKey(sshPrivKeyPath)
-	if err != nil {
-		return err
-	}
-
-	if fileInfo.Size() == 0 {
+		m.log.WithError(err).Error("error stat'ing file containing private key")
+		return "", err
+	} else if fileInfo.Size() == 0 {
 		m.log.Warn("cannot gather logs/tarball as ssh private key file is empty")
```

Comment:
This log entry should mention initializing SSH agent rather than gathering logs.

Reply:
done